settings.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+from dotenv import load_dotenv
+from pydantic.v1 import BaseSettings
+
+load_dotenv()
+
+
+class Settings(BaseSettings):
+    PROJECT_NAME: str = os.getenv("PROJECT_NAME")
+    DATABASE_URL: str = os.getenv("DATABASE_URL")
```

Comment:
Ensure that the `DATABASE_URL` environment variable is set in your environment. If it's not set, the application may fail to connect to the database.